tools/chromium_src/lit_mangler/mangle.test.ts

```diff
@@ -38,10 +38,26 @@ describe('mangled files should have up to date snapshots', () => {
             const manglerPath = name.replace('.lit_mangler.ts', '')
                 .replace(`chromium_src/`, '')
             const baseName = path.basename(manglerPath)
+            const originalDir = path.basename(path.join(manglerPath, '..'))
 
-            const mangledPath = path.relative(root, path.join(genPath, manglerPath.replace(baseName, 'preprocessed/' + baseName)))
+            const assumedMangledPath = path.relative(root, path.join(genPath, manglerPath.replace(baseName, 'preprocessed/' + baseName)))
+
+            let mangledPath = assumedMangledPath
             const originalPath = path.join('..', manglerPath)
 
+            let attempts = 0
+            while (attempts < 30 && !fs.existsSync(mangledPath)) {
```

Comment:
I think rather than limiting the number of attempts, lets just stop it from searching past the `gen` folder.